ai-server/app/api/v1/endpoints.py

```
@@ -0,0 +1,27 @@
+class PromptRequest(BaseModel):
+    prompt: str
+
+
+def get_musicgen_model():
+    if not hasattr(get_musicgen_model, "model"):
+        get_musicgen_model.model = MusicGen.get_pretrained('small')
+    return get_musicgen_model.model
+
+
+@router.post("/generate-music")
+async def generate_music(request: PromptRequest, model=Depends(get_musicgen_model)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)